src/vs/editor/common/controller/oneCursor.ts

```diff
@@ -553,6 +552,18 @@ export class OneCursor {
 	public getViewLineMaxColumn(lineNumber:number): number {
 		return this.viewModelHelper.viewModel.getLineMaxColumn(lineNumber);
 	}
+	public getViewLineMinColumn(lineNumber:number): number {
+		return this.viewModelHelper.viewModel.getLineMinColumn(lineNumber);
+	}
+	public getViewLineCenterColumn(lineNumber:number): number {
```

Comment:
You have to be careful about this one. This does not work for lines containing bi-di or variable width character sizes.